tests/FakeItEasy.Tests/ReturnValueConfigurationExtensionsTests.cs

```diff
@@ -775,7 +775,7 @@ public void Curried_ReturnsLazily_returns_value_from_curried_function()
             // Assert
             var curriedFunction = Fake.GetCalls(config).Single().Arguments.Get<Func<IFakeObjectCall, int>>(0);
 
-            curriedFunction.Invoke(A.Dummy<IFakeObjectCall>()).Should().Be(currentValue);
+            curriedFunction!.Invoke(A.Dummy<IFakeObjectCall>()).Should().Be(currentValue);

```

Comment:
Is this necessary? Looking at the assignment, `curriedFunction` shouldn't be nullable

Reply:
`ArgumentCollection.Get<T>(int)` is `MaybeNull` because we do not have control over the values that will be passed to the fakes…

Reply:
Ah, yes.